package.json

```diff
@@ -54,7 +54,7 @@
     "dotenv": "5.0.1",
     "dotenv-webpack": "^1.5.5",
     "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-adapter-react-16": "npm:enzyme-react-adapter-future",
```

Comment:
More info about this can be found here https://github.com/airbnb/enzyme/pull/1513#issuecomment-397128309